src/platform/user/tests/authentication/utilities.unit.spec.js

```diff
@@ -169,37 +167,74 @@ describe('standaloneRedirect', () => {
 
   it('should return null when an application param is not provided', () => {
     global.window.location.search = '';
-    expect(standaloneRedirect()).to.be.null;
+    expect(authUtilities.standaloneRedirect()).to.be.null;
   });
 
   it('should return null when an application redirect is not found', () => {
     global.window.location.search = '?application=unmappedapplication';
-    expect(standaloneRedirect()).to.be.null;
+    expect(authUtilities.standaloneRedirect()).to.be.null;
   });
 
   it.skip('should return a plain url when no `to` search query is provided', () => {
     global.window.location.search = '?application=myvahealth';
-    expect(standaloneRedirect()).to.equal(externalRedirects.myvahealth);
+    expect(authUtilities.standaloneRedirect()).to.equal(
+      authUtilities.externalRedirects.myvahealth,
+    );
   });
 
   it('should strip any CRLF characters from the "to" parameter', () => {
     global.window.location.search =
       '?application=myvahealth&to=/some/sub/route\r\n';
-    expect(standaloneRedirect()).to.equal(
-      `${externalRedirects.myvahealth}/some/sub/route`,
+    expect(authUtilities.standaloneRedirect()).to.equal(
+      `${authUtilities.externalRedirects.myvahealth}/some/sub/route`,
     );
   });
 });
 
 describe('loginAppUrlRE', () => {
   it('should resolve to a login app url', () => {
-    expect(loginAppUrlRE.test('/sign-in')).to.be.true;
-    expect(loginAppUrlRE.test('/sign-in/')).to.be.true;
-    expect(loginAppUrlRE.test('/sign-in/verify')).to.be.true;
-    expect(loginAppUrlRE.test('/sign-in/verify/')).to.be.true;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in')).to.be.true;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in/')).to.be.true;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in/verify')).to.be.true;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in/verify/')).to.be.true;
   });
   it('should not resolve to a login app url', () => {
-    expect(loginAppUrlRE.test('/sign-in-faq')).to.be.false;
-    expect(loginAppUrlRE.test('/sign-in-faq/')).to.be.false;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in-faq')).to.be.false;
+    expect(authUtilities.loginAppUrlRE.test('/sign-in-faq/')).to.be.false;
+  });
+});
+
+describe('login DOM ', () => {
```

Comment:
What are your thoughts on separating this into a new file?

Reply:
@AGarcia-Clarity I am not opposed to it. Do you have a recommendation of where you would prefer it?

Reply:
I think just create a `components/` directory inside where our current authentication tests live: `src/platform/user/tests/authentication/components` or where-ever else you want it.